Lib/glyphsets/test_strings.json

```diff
@@ -54,27 +42,35 @@
             {
                 "string": "010 080 030 020 050 060 070",
                 "lang": "en"
+            },
+            {
+                "string": "3+4=7 5>2 2x²−(8×7)÷9≈1",
+                "lang": "en"
+            },
+            {
+                "string": "ĿL·L ŀl·l",
```

Comment:
so if we have the lang tag, it should do the substitution by itself, therefore we would only need the string `L periodcentered L` and `l periodcentered l` and it should replace either by the Ldot or periodcentered.loclCAT, right?

Reply:
What I'm currently testing ;-). It seems that periodcentered only gets tranformed if there's another "l" at the end

```
hb-shape ~/Type/fonts/ofl/wixmadefortext/WixMadeforText\[wght\].ttf "l·l" --language ca
[l=0+247|periodcentered.loclCAT=1+182|l=2+247]
```

What we've currently got
```
hb-shape ~/Type/fonts/ofl/wixmadefortext/WixMadeforText\[wght\].ttf "l·" --language ca
[l=0+247|periodcentered=1+198]
```

Reply:
yes that makes sense

Reply:
Sorry, I'm being an idiot. We have it correct already.

Reply:
I think the string you've already got is great.

`"ĿL·L ŀl·l",`

The first character is Ldot and the next three chars compare whether the locl CAT feature result matches. I think this is spot on tbh,

Reply:
Ldot and ldot may appear as tofu because we don't require them anymore. We would expect these codepoints only in monospace and eventually cursive fonts

Reply:
ok cool, I'll drop them from the string.

Reply:
Ok, it's working as expected on WixMadeFor.

<img width="414" alt="Screenshot 2023-11-07 at 12 08 35" src="https://github.com/googlefonts/glyphsets/assets/7525512/c5ce7601-91ed-471e-aaea-7a9744c45a0d">

I'll remove the precomposed chars.